setup.php

```diff
@@ -743,7 +745,9 @@ function cloneClicked() {
 SOURCESITEBOT;
                             echo $source_site_bot ."\r\n";
                         }
-                        $randomusername = chr(rand(65, 90)) . chr(rand(65, 90)) . chr(rand(65, 90)) . "-admin-" . rand(0, 9) . rand(0, 9);
+                        $randomusername = chr(rand(65, 90)) . chr(rand(65, 90)) . chr(rand(65, 90)) . "-admin-" . rand(0, 9) . rand(0, 9);                                                // App Based TOTP secret
+                        $randomsecret = chr(rand(0, 90)) . chr(rand(0, 90)) . chr(rand(0, 90)) . chr(rand(0, 90)) . chr(rand(0, 90)) . chr(rand(0, 90));
```

Comment:
TODO for me:
Make cryptographically secure. Also looks like encoding should only be with base32 variables for the secret key. Still researching this.